api/HTMLMediaElement.json

```diff
@@ -206,10 +232,24 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "24",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
```

Comment:
I didn't think Opera used our flags.

Reply:
They do. It's under opera://flags and I've just confirmed that enabling experimental web platform features there causes the APIs touched in this PR to be exposed.

In general, I just mirror the data to Opera if the I'm familiar with the change in Chromium, so I know it's not something especially likely to be enabled or disabled by embedders. It's a hassle to maintain the flag data, but that's something to sort out in https://github.com/mdn/browser-compat-data/issues/3318.